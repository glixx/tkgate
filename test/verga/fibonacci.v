module top;

  integer i;
  reg [31:0] m[0:1024];

  initial
    begin
      $display("starting Fibonacci");

      m[0] = 1;
      m[1] = 1;
      for (i = 2;i < 20;i = i + 1)
	begin
	  m[i] = m[i-1] + m[i-2];
	end
      for (i = 0;i < 20;i = i + 1)
	begin
	  $display("m[%d] = %d",i,m[i]);
	end
    end

endmodule
