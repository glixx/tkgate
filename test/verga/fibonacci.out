starting Fibonacci
m[0] = 1
m[1] = 1
m[2] = 2
m[3] = 3
m[4] = 5
m[5] = 8
m[6] = 13
m[7] = 21
m[8] = 34
m[9] = 55
m[10] = 89
m[11] = 144
m[12] = 233
m[13] = 377
m[14] = 610
m[15] = 987
m[16] = 1597
m[17] = 2584
m[18] = 4181
m[19] = 6765
